modules/Data.py

```diff
@@ -114,6 +114,11 @@ def update_conversion_rates(output_currency, json_output_enabled):
                 log.log_error("Failed to find the exchange rate for outputCurrency {0}!".format(output_currency))
                 log.log_error("Make sure that {0} is either traded on Poloniex or supported by blockchain.info: {1}"\
                               .format(output_currency, "https://blockchain.info/api/exchange_rates_api"))
+                log.log_error("Using BTC as the output currency".format(url))
+                update_conversion_rates('BTC', json_output_enabled)
```

Comment:
I don't like calling the function again as it does a call to polo backend and updates all rates.  
However, I have an idea, we should set the BTC out by default by changing this:
```
         if output_currency == 'BTC':
             output_currency_found = True
             log.updateOutputCurrency('highestBid', '1')
             log.updateOutputCurrency('currency', output_currency)
```
to:
```
         log.updateOutputCurrency('highestBid', '1')
         log.updateOutputCurrency('currency', 'BTC')
         if output_currency == 'BTC':
             output_currency_found = True
```